Week3/app/src/main/java/com/kulloveth/businesscard/MainActivity.kt

```diff
@@ -0,0 +1,221 @@
+package com.kulloveth.businesscard
+
+
+import android.content.Intent
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.core.app.ShareCompat
+import androidx.core.content.ContextCompat
+import com.kulloveth.businesscard.databinding.ActivityMainBinding
+import java.util.*
+
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
```

Comment:
Nice work exploring the view binding. I know this is becoming more prevalent in Android all the time and can be such a useful feature.